give.php

```diff
@@ -564,6 +582,14 @@ private function includes() {
 			require_once GIVE_PLUGIN_DIR . 'includes/deprecated/deprecated-actions.php';
 			require_once GIVE_PLUGIN_DIR . 'includes/deprecated/deprecated-filters.php';
 
+			require_once GIVE_PLUGIN_DIR . 'includes/forms/route.php';
+			require_once GIVE_PLUGIN_DIR . 'templates/form-styles/elegent/actions.php';
+			require_once GIVE_PLUGIN_DIR . 'templates/form-styles/elegent/filters.php';
+			require_once GIVE_PLUGIN_DIR . 'src/Form/Themes.php';
+			require_once GIVE_PLUGIN_DIR . 'src/Form/Theme.php';
```

Comment:
These are classes, right? Shouldn't these be autloaded?

Reply:
@JasonTheAdams Added by mistake

Reply:
https://github.com/impress-org/givewp/compare/f585d0efd09a...bca8812a5a25